system-parachains/asset-hubs/asset-hub-polkadot/src/ah_migration/call_filter.rs

```diff
@@ -70,54 +70,54 @@ pub fn call_allowed_status(call: &<Runtime as frame_system::Config>::RuntimeCall
 	const OFF: bool = false;
 
 	let during_migration = match call {
-		AhMigrator(..) => ON,
-		AhOps(..) => OFF,
-		AssetConversion(..) => OFF,
+		AhMigrator(..) => ON, // required for the migration, only permissioned calls
+		AhOps(..) => OFF,     // Not needed during the migration
+		AssetConversion(..) => ON, // no reason to disable it, just convenience
 		AssetRate(..) => OFF,
-		Assets(..) => OFF,
-		Balances(..) => OFF, // TODO: @muharem enable
+		Assets(..) => ON,   // no reason to disable it, just convenience
+		Balances(..) => ON, // no reason to disable it, just convenience
 		Bounties(..) => OFF,
 		ChildBounties(..) => OFF,
 		Claims(..) => OFF,
-		CollatorSelection(..) => OFF, /* TODO: @muharem maybe disable them since staking is also */
-		// disabled?
+		CollatorSelection(..) => OFF, // TODO: maybe disable them since staking is also disabled?
 		ConvictionVoting(..) => OFF,
 		CumulusXcm(..) => OFF, /* Empty call enum, see https://github.com/paritytech/polkadot-sdk/issues/8222 */
 		FastUnstake(..) => OFF,
-		ForeignAssets(..) => ON,
+		ForeignAssets(..) => ON, // no reason to disable it, just convenience
 		Indices(..) => OFF,
 		MultiBlockElection(..) => OFF,
 		MultiBlockElectionSigned(..) => OFF,
 		MultiBlockElectionUnsigned(..) => OFF,
 		MultiBlockElectionVerifier(..) => OFF,
-		MessageQueue(..) => ON, // TODO: @muharem think about this
+		MessageQueue(..) => ON, // contains non-permissioned service calls
 		Multisig(..) => OFF,
-		Nfts(..) => ON,
+		Nfts(..) => ON, // no reason to disable it, just convenience
 		NominationPools(..) => OFF,
-		ParachainInfo(..) => OFF, /* Empty call enum, see https://github.com/paritytech/polkadot-sdk/issues/8222 */
-		ParachainSystem(..) => ON, // Only inherent and root calls
-		PolkadotXcm(..) => ON,
-		PoolAssets(..) => ON,
+		ParachainInfo(parachain_info::Call::__Ignore { .. }) => OFF, // Has no calls
+		ParachainSystem(..) => ON,                                   /* Only inherent and root */
+		// calls
+		PolkadotXcm(..) => ON, // no reason to disable it, just convenience
+		PoolAssets(..) => ON,  // no reason to disable it, just convenience
 		Preimage(..) => OFF,
 		Proxy(..) => OFF,
 		Referenda(..) => OFF,
-		Scheduler(..) => ON,
+		Scheduler(..) => ON, // only permissioned service calls
 		Session(..) => OFF,
 		Staking(..) => OFF,
```

Comment:
Just to clarify, `Staking` will be OFF pre migration (post AHM RU) and during migration. Only switching ON post migration. Correct?

Reply:
Actually we currently dont have a filter for *before* the migration, only during and after. So we need to change it and also include a *before* filter.